docs/source/examples/checks/vision/performance/confusion_martix.ipynb

```diff
@@ -0,0 +1,9265 @@
+{
```

Comment:
<p>There's supposed to be an available model in datasets/.../ mnist.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchecks/deepchecks/pull/828/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>